### PR TITLE
Simple object detection, avoids walls

### LINKED ISSUE #20: Check for wall
- Make a function that checks if there is a wall in front of the robot

### PR DESCRIPTION
**Pull request by leonpx**
Closes #20.

### CODE REVIEW

File: robot/main.py [hunk 1]
```diff
@@ -17,4 +17,27 @@
...
+robot = DriveBase(left_motor, right_motor, wheel_diameter=50, axle_track=90)
+
+while True:
+    # Begin driving forward at 200 millimeters per second.
+    robot.drive(200, 0)
+
+    # Wait until an obstacle is detected. This is done by repeatedly
+    # doing nothing (waiting for 10 milliseconds) while the measured
+    # distance is still greater than 300 mm.
+    while obstacle_sensor.distance() > 300:
```
Comment (Kax-y) [inline robot/main.py]: Move this while loop into a new function "check_for_wall" or "obstacle_ahead" or something like that.

File: robot/main.py [hunk 2]
```diff
@@ -17,4 +17,27 @@
...
+#ev3.speaker.beep()
+left_motor = Motor(Port.A)
+right_motor = Motor(Port.B)
+obstacle_sensor = UltrasonicSensor(Port.S4)
+
+#motor.run_time(360, 10000, then=Stop.HOLD, wait=True)
+
+robot = DriveBase(left_motor, right_motor, wheel_diameter=50, axle_track=90)
+
+while True:
```
Comment (Kax-y) [inline robot/main.py]: Put this whole while loop into a test function and then run it in "if __name__ == 'main':"

**Review by Kax-y — CHANGES_REQUESTED** [review]

File: robot/control.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+
+def check_for_obstacle(obstacle_sensor):
```
Comment (Kax-y) [inline robot/control.py:2]: Shouldn't this function return something? For example True when it detects something and False otherwise?

**Review by Kax-y — DISMISSED** [review]
Looks good just one minor detail that you could optionally fix

**Review by 00oskpet — DISMISSED** [review]

**Pull request merged**